frontends/sdl2/display.lisp

```diff
@@ -306,3 +306,10 @@
   (utils:create-texture (display-renderer display)
                         (* width (display-char-width display))
                         (* height (display-char-height display))))
+
+(defmethod lem-if:set-font-with-implementation ((implementation lem-sdl2/sdl2:sdl2) font-name)
```

Comment:
I'm not sure if that specialization should be in `sdl2/display.lisp` or `sdl2/main.lisp` where most of lem-if specializations are.

Reply:
I think lem-if should be implemented in all frontends.
Otherwise, I think it would be fine to use it as a symbol within sdl2 for the time being.
What do you think?

Reply:
That said, I feel like this is fine.